security/pdp/sample-xacml/src/main/java/ddf/security/pdp/xacml/SampleXACML.java

```diff
@@ -25,71 +25,104 @@
 import org.slf4j.LoggerFactory;
 
 public class SampleXACML {
```

Comment:
Is this supposed to be in here? This looks like it may have been carried over from one of your other commits.